kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/PodLogExample.java

```diff
@@ -15,42 +15,38 @@
  */
 package io.fabric8.kubernetes.examples;
 
-import io.fabric8.kubernetes.client.Config;
-import io.fabric8.kubernetes.client.ConfigBuilder;
 import io.fabric8.kubernetes.client.DefaultKubernetesClient;
 import io.fabric8.kubernetes.client.KubernetesClient;
-import io.fabric8.kubernetes.client.KubernetesClientException;
 import io.fabric8.kubernetes.client.dsl.LogWatch;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@SuppressWarnings("java:S106")
 public class PodLogExample {
 
   private static final Logger logger = LoggerFactory.getLogger(PodLogExample.class);
 
   public static void main(String[] args) {
     if (args.length < 1) {
-      System.out.println("Usage: podName [master] [namespace]");
+      System.out.println("Usage: podName [namespace]");
```

Comment:
I left some of the examples with System.out|err (those annotated with `@SuppressWarnings("java:S106")`), since the output stream is used later on either to print stuff (logs in this case), or as a parameter (see Exec examples).